smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/DefaultConfigTest.java

```diff
@@ -89,12 +89,14 @@ private MapBasedConfig getKafkaSinkConfigForMyAppProcessingData(String topicOut,
         Map<String, Object> config = new HashMap<>();
         config.put(prefix + "connector", KafkaConnector.CONNECTOR_NAME);
         config.put(prefix + "topic", topicOut);
+        config.put(prefix + "tracing-enabled", false);
```

Comment:
That is a bit cumbersome and error-prone.
Is there something we can do in the base class?

Reply:
Will see if something can be done

Reply:
I was thinking I could detect whether it was set in `MapBasedConfig` and then add it set to false in `write()` if it wasn't.

But I'm not sure that's going to be easy given the different key prefixes and an unknown number of them.